packages/landing/src/components/map.tsx

```diff
@@ -52,7 +52,7 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
     const cfg = Config.map;
     const tileGrid = getTileGrid(cfg.tileMatrix.identifier);
     const style = tileGrid.getStyle(cfg.layerId, cfg.style);
-    const location = cfg.transformedLocation;
+    const location = locationTransform(cfg.location, Config.map.tileMatrix, GoogleTms);
```

Comment:
everything else is using `cfg` here so `cfg.tileMatrix`

Reply:
Good pick.